src/triggers/default_triggers.py

```diff
@@ -24,6 +24,7 @@
             workers_count=0,
             high_availability_mode=consts.HighAvailabilityMode.NONE,
             user_managed_networking=True,
+            platform=consts.Platforms.NONE,
```

Comment:
hmm, but it means we changed the API
if we have a customer using REST API (or maybe even UI) it means he needs to add this platform

Is it possible making assisted-service infer none-platform when on SNO? (I guess like before)

Reply:
No, because we are setting `baremetal` as default platform. Baremetal and UMN enabled are not a valid input anymore.
Maybe setting platform to python None (null) will be better than changing the platform to none (although that in the service it will be none anyway)